howto/instance/create.md

```diff
@@ -0,0 +1,76 @@
+You can either launch an instance with `amc launch` or initialise an instance with the `amc init` command for a registered application or image (see [Application instances vs. raw instances](https://discourse.ubuntu.com/t/17763#application-vs-raw)), by using the `amc` tool or through another service over the REST API that the AMS service provides. The `amc init` command only creates the instance, while the `amc launch` command creates and starts it.
+
+By default, the instance will run headless. See [How to access a instance](https://discourse.ubuntu.com/t/17772) for instructions on how to access it for debugging purposes, and [About application streaming](https://discourse.ubuntu.com/t/streaming-android-applications/17769) for information about the streaming stack.
+
+[note type="information" status="Note"] The following examples use `amc launch`, but you can use `amc init` in the same way.[/note]
+
+<a name="application-instances"></a>
+## Launch application instances
+
+Launching an instance for a registered application can be achieved with the following command:
+
+    amc launch <application_id>
+
+If you want to launch the application in a virtual machine, use the `--vm` flag with the `amc launch` command:
+
+    amc launch --vm <application_id>
```

Comment:
No, you don't need --vm when you specify an app name or id as the information is encoded in the app whether you will create a VM or a container. There is no further need to specify this.

Reply:
Maybe you can add what Simon wrote instead, so it's perfectly clear for the user that the instance type is encoded in the application.